pallets/rewards/src/tests/currency_movement.rs

```diff
@@ -83,7 +74,10 @@ macro_rules! currency_movement_tests {
 					assert_ok!($pallet::attach_currency(DOM_1_CURRENCY_M, GROUP_B)); // MOVEMENT HERE!!
 					assert_ok!($pallet::deposit_stake(DOM_1_CURRENCY_M, &USER_A, STAKE_M));
 
-					has_currencies(USER_A, &[(DomainId::D1, CurrencyId::M)]);
+					assert_eq!(
```

Comment:
The currency movement does not affect the number of currencies in an account, so maybe is better to move this check to its own test in `rewards/src/tests/common.rs` under `currency_common_tests` macro. To leave these tests free of this.

That test should associate different currencies to an amount and check the correct list of currencies in that amount.

Reply:
Thank You!